includes/class-newspack-newsletters.php

```diff
@@ -780,6 +799,34 @@ public static function default_title( $post_title, $post ) {
 		return $post_title;
 	}
 
+	/**
+	 * Handle custom Newsletter styling when viewing the newsletter as a public post.
+	 */
+	public static function public_newsletter_custom_style() {
+		$post = get_post();
+		if ( $post && self::NEWSPACK_NEWSLETTERS_CPT === $post->post_type ) {
+			$font_header      = get_post_meta( $post->ID, 'font_header', true );
+			$font_body        = get_post_meta( $post->ID, 'font_body', true );
+			$background_color = get_post_meta( $post->ID, 'background_color', true );
+			?>
+				<style>
+					.main-content {
+						background-color: <?php echo esc_attr( $background_color ); ?>;
```

Comment:
Yes we should.

Reply:
Added in b59ce0e07c7d6db09aa2303e89369993da381c88

Reply:
Since Newspack theme uses `32px` for its margins, we should use the same here. Also probably want to avoid padding top and bottom since the blocks add a margin top and bottom already. So I suggest `padding: 0 32px;`

Reply:
Updated in 1d136bed133d3e3aa25f1c2143195ed30f30af55